### ISSUE TITLE
How to Combine a Multi-Texture PLY File into a Single-Texture PLY File to run?

### ISSUE THREAD

**Issue by Xtian-hub** [tudelft3d/SUMS-Semantic-Urban-Mesh-Segmentation-public#22]
I have downloaded data that contains OBJ and multiple texture images. I can convert it to a PLY file, but it still contains multiple texture images. This makes the data too large and difficult to run when using the tool. Is there a method to merge and compress the texture images in this kind of data?
like one to two as follow:
![a7a7cabd6a65587627db9fa760069ec](https://github.com/user-attachments/assets/c592e4f3-d81f-4a78-8af6-a07a53fdfb69)
![92925d098ec5429bcf85745bf111b44](https://github.com/user-attachments/assets/f92abf73-4960-4532-af1d-4b31682e31da)
@WeixiaoGao

**Comment by WeixiaoGao** [comment]
Hi, it's true that the current version of the annotation tool is insufficient to handle multiple textures and the large file size of the mesh. I will try to improve it in the future. As for merging textures, I am not aware of any tools that can merge multiple textures into one. I think one possible way is to use the original images to generate a new texture image for your mesh data.

**Comment by Xtian-hub** [comment]
您好，我是耿晓天我已经收到了您的来信，稍后我会回复，谢谢?7?8(?1?1ω?1?1)?1?7

**Comment by Xtian-hub** [comment]
Hello, Dr. Gao, thank you for your response. I’m very much looking forward to the further improvements you and your team will make to the tool. In addition, I have tested the model on my local data and found that it performs well in recognizing green tree species. However, due to seasonal variations, I noticed that some trees that turn yellow in autumn were mistakenly identified as buildings. Could you please advise on how I can fine-tune your pre-trained model so that it can accurately segment trees or other vegetation across different seasons?
![image](https://github.com/user-attachments/assets/f976baec-5437-4b30-9160-ba5f1bfa0f9e)
![326f093224d220c8c8462e66dfa8f20](https://github.com/user-attachments/assets/99b9d971-d573-463c-abf8-f88538fbfbcb)
Additionally, I have downloaded your pre-trained model. Could you clarify if this model is based on point cloud segmentation or another type of neural network model? If I wish to add other segmentation models, are there specific structural requirements for the files within the compressed package? @WeixiaoGao

**Comment by WeixiaoGao** [comment]
Hi Xiaotian, in this repository, the pre-trained model is a machine learning model based on meshes. The feature 'greenness' contributes significantly to identifying trees; however, if the vegetation does not appear very green, it may lead to errors. There is little you can do to overcome this by only tuning the parameters. If you want to use an ML approach, you might need to develop some new handcrafted features to address this issue. However, this might not be a very popular approach currently. The competition method is based on sampled point clouds using deep learning methods. You can use them to train your own model for better results. Our tool can also be used to generate sampled point clouds and parse the labels back to the meshes.

**Comment by Xtian-hub** [comment]
Thank you very much for your response. I will try to use a deep learning model to train and predict on the sampled point clouds. From your response, I understand that if the deep learning model successfully segments the point clouds, I can import the segmented point clouds and labels back into SUMS to apply the labels to the mesh. Is my understanding correct?

**Comment by WeixiaoGao** [comment]
Yes, that's correct.

**Comment by Xtian-hub** [comment]
Thank you very much for your response. It has given me great ideas, and I will make further attempts.
